README.md

```diff
@@ -2,9 +2,9 @@
 
 [![Build Status](https://travis-ci.com/dashevo/drive.svg?token=Pzix7aqnMuGS9c6BmBz2&branch=master)](https://travis-ci.com/dashevo/drive)
 
-> Dash network data storage backend service.
+> Distributed, decentralized application storage on the Dash p2p network.
```

Comment:
Drive is not distributed yet :).

Decentralized off-chain data storage on the Dash p2p network. ?